backend/src/test/java/com/jujeol/RequestBuilder.java

```diff
@@ -161,18 +166,30 @@ private RequestSpecification requestWithDocument() {
 
         private class UserHelper {
             private boolean userFlag;
+            private String token;
             private TestMember member;
 
             public UserHelper() {
                 this.userFlag = false;
+                this.token = "";
             }
 
             public void withUser(TestMember testMember) {
                 userFlag = true;
                 this.member = testMember;
             }
 
+            public void withUser(String token) {
+                userFlag = true;
+                this.token = token;
+            }
+
             public void addRequest(RequestSpecification requestSpec) {
+                if(token.length() != 0) {
+                    requestSpec.header("Authorization", "Bearer " + token);
+                    return;
+                }
+
                 if(userFlag) {
                     final TokenDto token = 
                             loginService.createToken(SocialProviderCodeDto.create(member.getMatchedCode(), ProviderName.TEST));
```

Comment:
```suggestion
private class UserHelper {
            private boolean userFlag;
            private String token;

            public UserHelper() {
                this.userFlag = false;
            }

            public void withUser(TestMember testMember) {
                userFlag = true;
                this.token = loginService.createToken(SocialProviderCodeDto.create(testMember.getMatchedCode(), ProviderName.TEST)).getAccessToken();
            }

            public void withUser(String token) {
                userFlag = true;
                this.token = token;
            }

            public void addRequest(RequestSpecification requestSpec) {
                if(userFlag) {
                    requestSpec.header("Authorization", "Bearer " + token);
                }
            }
        }
```

이렇게 전체적으로 바꾸면 더 이쁠 거 같아~!

Reply:
크헤헤 withUser(token) 아래 메서드를 위에서 사용하는 건뎅 메서드 넣어주는 값은 분리해도 좋을 것 같구!